scripts/scil_convert_surface.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to convert cortical surface (vtk or freesurfer).
```

Comment:
Remove cortical (can be anything right?)
Can you list the accepted file format?